controllers/PageController.php

```diff
@@ -420,32 +437,18 @@ public function actionEntry(int $id = null)
     }
 
     /**
-     * @param string|int $id
-     * @return WikiPage|null
+     * @inheritdoc
      */
-    private function getWikiPage($id): ?WikiPage
+    protected function getAccessRules()
```

Comment:
@marc-farre Out of curiosity: Is there a reason for moving this method?

Reply:
It's because of this setting on phpStorm on file saving:
![image](https://github.com/humhub/wiki/assets/23310825/29556fce-5fa8-4e92-a50e-b63a0fbae898)

What do you suggest? If you have a better setting, maybe I could apply it too to avoid code style changing when I edit HumHub files?

Reply:
We currently have no special rules for the order of methods/attributes. It would therefore be good to deactivate rules of this type.

We can add rules here in the future, but these should then also be stored in the PhpCsFixer Config and applied to the entire codebase.

See: https://github.com/humhub/humhub/blob/develop/.php-cs-fixer.php

--- 

For this PR now, we can leave anything as it is.